src/modules/notes/controllers/createNote.js

```diff
@@ -0,0 +1,11 @@
+import { PROVIDE_VALUES } from '../constants/index.js';
+
+export const createNote = (req, res) => {
+	if (!Object.keys(req.body).length) {
```

Comment:
Replace please this calculation with proper variable. 
For example: `noParameters` or `noProps`